Core/PoEMemory/Elements/InstanceManagerPanel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ExileCore.PoEMemory.Elements
+{
+    public class InstanceManagerPanel : Element
```

Comment:
what's an instance manager panel?

Reply:
It's the window that shows up when you press Ctrl while clicking portal/waypoint to create a new instance of an area.